docs/nominatim.openapi.json

```diff
@@ -299,6 +299,171 @@
+                    {
+                        "schema": {
+                            "type": "string"
+                        },
+                        "in": "query",
+                        "name": "json_callback",
+                        "description": "Wrap JSON output in a callback function (JSONP) i.e. <string>(<json>). Only has an effect for JSON output formats."
+                    },
+                    {
+                        "schema": {
+                            "type": "integer",
+                            "maximum": 1,
+                            "minimum": 0
+                        },
+                        "in": "query",
+                        "name": "addressdetails",
+                        "description": "Include a breakdown of the address into elements."
+                    },
+                    {
+                        "schema": {
+                            "type": "integer",
+                            "maximum": 1,
+                            "minimum": 0
+                        },
+                        "in": "query",
+                        "name": "extratags",
+                        "description": "Include additional information in the result if available, e.g. wikipedia link, opening hours."
+                    },
+                    {
+                        "schema": {
+                            "type": "integer",
+                            "maximum": 1,
+                            "minimum": 0
+                        },
+                        "in": "query",
+                        "name": "namedetails",
+                        "description": "Include a list of alternative names in the results. These may include language variants, references, operator and brand."
+                    },
+                    {
+                        "schema": {
+                            "type": "string"
+                        },
+                        "in": "query",
+                        "name": "accept-language",
+                        "description": "Preferred language order for showing search results, overrides the value specified in the [Accept-Language HTTP header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language). Either use a standard RFC2616 accept-language string or a simple comma-separated list of language codes."
+                    },
+                    {
+                        "schema": {
+                            "type": "string"
+                        },
+                        "in": "query",
+                        "name": "countrycodes",
+                        "description": "Limit search results to one or more countries. `<countrycode>` must be the [ISO 3166-1alpha2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) code. Each place in Nominatim is assigned to one country code based on OSM country boundaries. In rare cases a place may not be in any country at all, for example, in international waters."
```

Comment:
There is no mention to `countrycodes` in the [official docs](https://nominatim.org/release-docs/develop/api/Lookup/#endpoint)...

```suggestion
```